src/modules/ie-objects/helpers/encode-search-term.ts

```diff
@@ -0,0 +1,10 @@
+import { ES_DIGIT_PREFIX } from '../elasticsearch/elasticsearch.consts';
+
```

Comment:
REQ: please add some comments to further elaborate what is being done here (+ some examples (not tests) but hardcoded examples).